.github/workflows/ci.yml

```diff
@@ -0,0 +1,165 @@
+name: CI
+run-name: CI (${{ github.event_name }}) for ${{ github.ref_name }}
+on:
+  # For config-pr-1-ci.yml
+  pull_request:
+    branches:
+      - 'release-*'
+      - 'dev-*'
+    paths-ignore:
+      # These are ignored because they don't have anything to do with the model itself
+      - .github/**
+      - tools/**
+      - doc/**
+      - config/**
+      - .*
+      - README.md
+  # For config-pr-2-confirm.yml
+  issue_comment:
+    types:
+      - created
+      - edited
+  # For config-pr-3-bump-tag.yml
+  create:
+  push:
+    branches:
+      - 'release-*'
+    paths:
+      - 'metadata.yaml'
+
+jobs:
+  ###########
+  # PR jobs #
+  ###########
+  pr:
+    name: PR
+    if: github.event_name == 'pull_request' && github.repository != 'ACCESS-NRI/model-configs-template'
+    uses: access-nri/model-config-tests/.github/workflows/config-pr-1-ci.yml@main
+    with:
+      additional-artifact-content-paths: |
+        archive/output000/MOM_parameter_doc.*
```

Comment:
`output000` is used in four locations. Should this be a global environment variable with a meaningful name. e.g. `PAYU_OUTPUT_DIR`.

Reply:
We can certainly do that for the other cases where it crops up, but we can't use the `env` context in a `with` section :(

Reply:
Done as much as I could in https://github.com/ACCESS-NRI/access-om3-configs/pull/122/commits/d2a6b425ef1d2f583714aed208d2e9c31d477b61